app/views/trainees/review_draft/show.html.erb

```diff
@@ -128,6 +128,15 @@
             classes: "school-details",
             status: "not started",
           )
+
+        component.slot(
```

Comment:
We already have a slot for this! It needs to be linked from the 'Schools' section above. As per the prototype. You'll see it's already wrapped in `if @trainee.requires_schools?` so that it doesn't show for non school direct trainees.

Reply:
updated